components/servo/proxies.rs

```diff
@@ -0,0 +1,44 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+use std::sync::atomic::{AtomicBool, Ordering};
+use std::sync::Arc;
+
+use compositing_traits::ConstellationMsg;
+use crossbeam_channel::{SendError, Sender};
+
+#[derive(Clone)]
+pub(crate) struct ConstellationProxy {
```

Comment:
We can open separate PR to add this type already. Btw could we use the convention of sender/receiver? It's easier to understand who send to who in this way.